src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -95,8 +98,39 @@ open class KtlintPlugin : Plugin<Project> {
         // Add the args to enable verbose and debug mode.
         if (extension.verbose) runArgs.add("--verbose")
         if (extension.debug) runArgs.add("--debug")
+        addReporter(extension, runArgs)
     }
 
+    private fun addReporter(extension: KtlintExtension, runArgs: MutableSet<String>) {
+        if (isReportAvailable(extension.version)) {
+            runArgs.add("--reporter=${extension.reporter}")
+        }
```

Comment:
Log a `debug` message in the `else` case. It isn't really a warning but it may help someone trying to debug why their isn't a report showing up.